### ISSUE TITLE
Disconnecting from game when changing any stage of a quest and a couple other conditions as well

### ISSUE THREAD

**Issue by Malicor56** [Azilroka/ProjectAzilroka#100]
Hello, I'm not seeing any other complaints about this, so I am assuming that there must be some kind of addon conflict occurring here.

I disconnect each time that I change the stage of a quest. I will pick up a quest, and then get disconnected, I will kill a monster for that quest and then get disconnected, I'll loot the body from the kill after getting logged back in to pick up and item for the quest and get disconnected.

I had noticed some other conditions that led to disconnects as well, I don't remember everything for it, I'll edit this and add it in when I do, but I just wanted to get this posted that way I might bring it to your attention and get some help on it. 

Thank you for this addon Azilroka! I love it!

**Comment by Azilroka** [comment]
Try disabling Quest Sounds.. Unless Blizzard changed something with PlaySound.. it should be fine.

**Comment by Malicor56** [comment]
I had considered this as well, as it seems to be the only "Quest" related feature. I am running some tests currently after having disabled the "Master Experience" and I haven't disconnected since. However, I have a couple of bug handler addons, bugswatter, bugsack, and an audio queue error alert addon, I was still getting audio queues indicating errors were occurring, but no more auto disconnects! yay! I'll post further updates as they come

**Comment by Azilroka** [comment]
Well Master Experience does send data over bnet and it's been broken for a bit.

**Comment by Malicor56** [comment]
So, it appears that the issue was Master Experience, and the "Broker LDB" That is probably because I am using titan panel or something like that and it is conflicting. I was hoping that it wasn't iFilger, as I wanted to experiment with that module and see what it's all about, I don't really know anything about the iFilger module, I just glanced over it.

So, TL;DR, I turned off the Master Experience that stopped the disconnects, and then I shut off the Broker LDB and that stopped the random continuous stream of errors (conflicting from my titan panel addon probably.)

Thank you again Azilroka! I love your work here buddy!
